resources/views/processes/index.blade.php

```diff
@@ -65,7 +65,7 @@
               <span aria-hidden="true">&times;</span>
             </button>
           </div>
-          <div class="modal-body" v-if="processCategories">
+          <div class="modal-body" v-if="hasProcessCategories">
```

Comment:
Cant we just check the length of the array in the v-if? I don't think we need all of this additional code...

Reply:
processCategories is an object so it does not have a length property. The only way to check the length of an object is by counting its keys with `Object.keys(obj)`. The original implementation in https://github.com/ProcessMaker/bpm/pull/1106 was broken. It probably couldn't be tested because you can not delete the seeded category unless you delete the process from the db